README.md

```diff
@@ -60,9 +60,10 @@ Here is [how to contribute](./contributing.md).
 * [Pyethereum](https://github.com/ethereum/pyethereum) in Python
     - Another client with probably the best readable EVM implementation ([ethereum/vm.py](https://github.com/ethereum/pyethereum/blob/develop/ethereum/vm.py))
 * [Py-EVM](https://github.com/pipermerriam/py-evm) in Python
-    - An alternate Python implementation designed to be highly configurable and modular.
+    - An alternate Python implementation designed to be highly configurable and modular and compliant with the Ethereum test suite (while Pyethereum isn't).
```

Comment:
I don't mention other clients in one client's description.

Reply:
I disagree here, but if you want to maintain your version, then feel free to edit this yourself.